kubernetes/values.yaml

```diff
@@ -34,12 +34,17 @@ securityContext:
 
 # See https://github.com/helm/charts/tree/master/stable/postgresql
```

Comment:
I think this comment need to be updated too.

Reply:
Great spotting, fixed thanks!